<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trumpet_widget</class>
 <widget class="QWidget" name="trumpet_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trumpet_widget</string>
  </property>
  <widget class="QGroupBox" name="widgetBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>1081</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Trumpet</string>
   </property>
   <widget class="QGroupBox" name="configBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Config:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Partial:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="partialCombo">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>1st</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2nd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3rd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4th</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5th</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Arpeggio</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="scaleCombo">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>C Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Db Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Eb Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>E Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gb Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ab Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bb Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B Major</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C# Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Eb Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>E Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F# Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G# Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bb Minor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B Minor</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Input History</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="inputList">
            <property name="font">
             <font>
              <family>Sans</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="removeButton">
              <property name="font">
               <font>
                <family>Sans</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="font">
               <font>
                <family>Sans</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="outputList">
          <property name="font">
           <font>
            <family>Sans</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="trumpetBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>641</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="trumpetLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets.qrc">:/trumpet/assets/trumpet_graphic.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="valve1">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="valve2">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="valve3">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="recordBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Playback:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="recordButton">
         <property name="font">
          <font>
           <family>Sans</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="font">
          <font>
           <family>Sans</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="recordBox_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Metronome:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QRadioButton" name="metronome">
         <property name="font">
          <font>
           <family>Sans</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBpm">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="inputHelpButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Fingering Help</string>
    </property>
   </widget>
   <zorder>trumpetBox</zorder>
   <zorder>configBox</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>recordBox</zorder>
   <zorder>recordBox_2</zorder>
   <zorder>inputHelpButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
